sale_partner_pricelist/models/res_partner.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Jarsa
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+import threading
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResPartnerr(models.Model):
+    _inherit = "res.partner"
+
+    allowed_pricelist_ids = fields.Many2many(
```

Comment:
I'm wondering if default values could come from a configuration on partner category to ease user day-to-day job ? Maybe in a ROADMAP ?

Reply:
I changed the behavior of the constraint, if the field is empty, it will allow all the pricelist.

With this change, we don't need to define a default value for this field